pkg/webui/console/views/device-general-settings/network-server-form/validation-schema.js

```diff
@@ -296,11 +303,13 @@ const validationSchema = Yup.object()
             return Yup.number().min(100000, Yup.passValues(sharedMessages.validateNumberGte))
           }),
           ping_slot_frequency: Yup.lazy(frequency => {
-            if (!Boolean(frequency) || !isClassB || mode === ACTIVATION_MODES.OTAA) {
+            if (!isClassB || (mode === ACTIVATION_MODES.OTAA && !Boolean(frequency))) {
               return Yup.number().strip()
             }
 
-            return Yup.number().min(100000, Yup.passValues(sharedMessages.validateNumberGte))
+            return Yup.number()
+              .min(100000, Yup.passValues(sharedMessages.validateNumberGte))
+              .required(sharedMessages.validateRequired)
```

Comment:
This is not working, I can still submit the form with an empty ping slot frequency when class B is enabled (resulting in a submit error). This should be caught on validation level already. Please check what's the issue there.